website/docs/components/button-set/partials/code/how-to-use.md

```diff
@@ -1,8 +1,42 @@
 ## How to use this component
 
+The basic invocation requires two or more buttons to be provided as children:
+
 ```handlebars
 <Hds::ButtonSet>
   <Hds::Button @text="Submit" type="submit" />
   <Hds::Button @text="Cancel" @color="secondary" @href="https://hashicorp.com" />
 </Hds::ButtonSet>
 ```
+
+### Equal width buttons
+
+If you want to have buttons with equal width, apply a width (via inline style or via CSS class) to the `ButtonSet` container, and the argument `@isFullWidth={{true}}` to the `Button` elements:
```

Comment:
Would it be more accurate to say `Button` components instead of elements?

Reply:
Yes, it would.